tools/azsdk-cli/Azure.Sdk.Tools.Cli/Configuration/RequirementsV1.json

```
@@ -0,0 +1,111 @@
+            }
+        ],
+        "javascript": [
+            {
+                "requirement": "pnpm",
+                "check": ["pnpm", "--version"],
+                "instructions": ["npm install -g pnpm", "Run pnpm install in the repo root."]
+            }
+        ],
+        "dotnet": [
+            {
+                "requirement": "dotnet",
+                "check": ["dotnet", "--version"],
+                "instructions": ["Install .NET SDK with version at least higher than defined in global.json file in the repo root."]
+            }
+        ],
+        "go": [
+            {
+                "requirement": "go",
+                "check": ["go", "version"],
+                "instructions": ["Download and install the Go version specified in globals.yml", "Ensure your GOPATH and GOROOT environment variables are set correctly", "Add GOROOT/bin to your PATH environment variable"]
+            },
+            {
+                "requirement": "go-junit-report",
+                "check": ["go-junit-report", "--version"],
+                "instructions": ["go install github.com/jstemmer/go-junit-report/v2@v2.1.0"]
+            },
+            {
+                "requirement": "gocov",
+                "check": ["gocov", "--version"],
+                "instructions": ["go install github.com/axw/gocov/gocov@v1.2.1"]
+            },
+            {
+                "requirement": "gocov-xml",
+                "check": ["gocov-xml", "--version"],
+                "instructions": ["go install github.com/AlekSi/gocov-xml@v1.1.0"]
+            },
+            {
+                "requirement": "gocov-html",
+                "check": ["gocov-html", "--version"],
+                "instructions": ["go install github.com/matm/gocov-html/cmd/gocov-html@v1.4.0"]
+            },
+            {
+                "requirement": "golangci-lint",
+                "check": ["golangci-lint", "--version"],
+                "instructions": ["https://golangci-lint.run/docs/welcome/install/"]
+            },
+            {
+                "requirement": "generator",
+                "check": ["generator", "--help"],
+                "instructions": ["go install github.com/Azure/azure-sdk-for-go/eng/tools/generator@latest"]
+            }
+        ],
+        "python": [
+            {
+                "requirement": "python >= 3.9",
+                "check": ["python", "--version"],
+                "instructions": ["Install Python from https://www.python.org/downloads/"]
+            },
+            {
+                "requirement": "azpysdk",
+                "check": ["azpysdk", "--help"],
+                "instructions": ["Ensure your virtual environment is activated", "python -m pip install eng/tools/azure-sdk-tools[build]", "Check that it was installed with `azpysdk --help`"]
```

```suggestion
                "instructions": ["Ensure your virtual environment is activated", "python -m pip install eng/tools/azure-sdk-tools[build,ghtools,sdkgenerator]", "Check that it was installed with `azpysdk --help`"]
```

Python need additional dependencies for sdk generation.